x-pack/plugins/security_solution/public/common/components/endpoint/agent_status.tsx

```diff
@@ -18,11 +18,7 @@ export const AgentStatus = React.memo(({ hostStatus }: { hostStatus: HostStatus
       data-test-subj="rowHostStatus"
       className="eui-textTruncate"
     >
-      <FormattedMessage
-        id="xpack.securitySolution.endpoint.list.hostStatusValue"
-        defaultMessage="{hostStatus, select, healthy {Healthy} unhealthy {Unhealthy} updating {Updating} offline {Offline} inactive {Inactive} unenrolled {Unenrolled} other {Unhealthy}}"
-        values={{ hostStatus }}
-      />
+      {getAgentStatusText(hostStatus)}
```

Comment:
Was this change intentional?

We use the code that was replaced in other places as well. We don't show every possible status from the Host - some all get aggregated under `Unhealthy`

Reply:
It was intentional - where is the code that aggregates statues under `Unhealthy` ?  My goal here was to consolidate the code that the previous AgentStatus component was using to ensure we display the status in a uniform way.  See this file: https://github.com/elastic/kibana/pull/135935/files#diff-a5a3673b7d60dda54e22d1b35ea57596210b6da1216abd79a6e8737706283a53R11

Reply:
yeah, I posted this one before I saw that we defined the function below.. sorry for the noise on this one.

Re: usages

I think endpoint list and endpoint details have similar code